### DISCUSSION TITLE
Responsive Tabs ?

### DISCUSSION THREAD

**Discussion by dukesx** [mantinedev/mantine#3291] [Feature requests]
## Why ?
Because its sleek, its SMART and tbh it's the most beautiful way to implement multiple tabs. 

### Use Cases
Blogs or CMS mostly, i mean at least i need this kind of responsiveness so that i can show users the tags they follow, but since this means ALOT of tabs ( 1 tab for each tag you follow) so a virtually scrollable tab like slider would be an awesome way , simple and efficient. 

![asdsdsd](https://user-images.githubusercontent.com/5988991/210850277-290ad2bc-b948-43e8-b4be-3eb4592f55ba.png)

**Comment by yogeshdofficial** [comment]
+1 for this. tabs wrapping to next line it not nice at all

**Comment by ghost** [comment]
This would be a great addition to Tabs :)

**Comment by dipiash** [comment]
I made an example with scrollable tabs. If it's interesting take a look at this repository: https://github.com/dipiash/mantine-scrollable-tabs. By default active the last tab and if you resize from large to small screen please refresh the page or select the visible tab. But if land on the page from scratch in mobile view it will be scrolled to active tab automatically

**Comment by enisdenjo** [comment]
[It's possible today with the Scroller component.](https://mantine.dev/core/scroller/#usage-with-tabs)

**Reply by dipiash** [reply]
Hi! I made an update. Added example with scroller

https://github.com/dipiash/mantine-scrollable-tabs
